library/stepper/scss/_mixins.scss

```
@@ -0,0 +1,37 @@
+@import './variables';
+
+@mixin amb-circular-stepper-classes() {
+  .CircularStepper {
+    border-radius: $amb-stepper-border-radius;
+    height: $amb-stepper-height;
+    width: $amb-stepper-width;
+    display: flex;
+    flex-direction: row;
+    background-image: conic-gradient(
+      amb-color-theme('secondary') 216deg,
+      amb-color-theme('secondary', 10) 216deg,
+      amb-color-theme('secondary', 10) 360deg
+    );
+
+    &::before {
+      content: '';
+      position: absolute;
+      margin: amb-spacing('xsmall');
+      border-radius: $amb-stepper-border-radius;
+      height: calc(60px - 15px);
+      width: calc(60px - 15px);
+      background-color: white;
+      z-index: $amb-stepper-z-index;
+    }
+
+    &-progressBar {
+      border-radius: $amb-stepper-border-radius;
+      background: transparent;
+      text-align: $amb-stepper-inner-text-align;
+      margin: amb-spacing('base');
+      position: absolute;
+      z-index: 100;
+      font-weight: amb-font-weight('medium');
```

@cakahlul 
- Ralat lul, kemarin gw kira ini class untuk progress barnya, ternyata hanya untuk indikatornya ya. Kalau begitu minta tolong diubah lagi nama stylerulenya. Mungkin jadi `&-indicator`?
- Terus disini gw ngeliat kejanggalan untuk absolute centernya, pas gw coba gedein size progess barnya, malah dia nggak ditengah lul, jadi implementasi ini masih agak _hacky_. Saran gw pakai either flex method atau positional method:

Contoh absolute centering pake positional method.
```suggestion
.CircularStepper {
  position: relative; // <- tambahin position relative disini biar positional ancestornya refer kesini instead of document flow
 
  &-progressBar {
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    ...
  }
}
```